vbr/vbrCore/functions/yosh2009.m

```diff
@@ -0,0 +1,66 @@
+function [ VBR ] = yosh2009( VBR )
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  %
+  % [ VBR ] = yosh2009( VBR )
+  %
+  % calculates the conductivity of San Carlos Olivine Aggregate at 10GPa
+  % from temperature and water content
+  %
+  % Parameters:
+  % ----------
+  % VBR    the VBR structure
+  %
+  % Output:
+  % ------
+  % VBR    the VBR structure, with VBR.out.electric.yosh2009_ol structure
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+  % read in eletric parameters
+  ele = VBR.in.electric.yosh2009_ol;
```

Comment:
See my comment above -- as `Params_Electric` is currently written, this line should be 
```
  ele = VBR.in.electric.yosh2009.yosh2009_ol;
```
If you remove the nested structure it would just be 

```
 ele = VBR.in.electric.yosh2009;
```